addons/account/static/src/helpers/account_tax.js

```diff
@@ -486,7 +486,7 @@ export const accountTaxHelpers = {
             }
         }
 
-        if (skipped.length > 0) {
+        if (skipped.size > 0) {
```

Comment:
not related to task? should be in another commit?

Reply:
@qdp-odoo I moved it to a separated commit. However, this code is used to handle the case in which a python tax returns a `None` tax amount meaning it just need to be ignore. Since python taxes were not usable before this commit, this "bug" wasn't really an issue before.